backends/ebpf/codeGen.cpp

```diff
@@ -27,7 +27,9 @@ void CodeGenInspector::substitute(const IR::Parameter* p, const IR::Parameter* w
 { substitution.emplace(p, with); }
 
 bool CodeGenInspector::preorder(const IR::Constant* expression) {
-    builder->append(Util::toString(expression->value, 0, false));
+    // Until EBPF/UBPF applications set the IR::Constant base to hex vs.
+    // decimal all emits will use hex.
+    builder->append(Util::toString(expression->value, 0, false, 16));
```

Comment:
Why is this necessary?

Reply:
Otherwise `4294963200` is emitted instead of `0xfffff000` for 

`headers.ipv6.ip_version_traffic_class_and_flow_label & ~4294963200`

Reply:
Why is that incorrect?

Reply:
Won't be incorrect.  A mask used with `&` is more readable as hex.  Any other constant would also be printed in hex, but that is fine too.  Note, even the bmv2 JSON format uses hex to emit any mask.

Reply:
This code is used for all constants, not just masks.
The right solution is to make sure that the constants created when expanding slices in the Lower pass actually use base 16.